src/fontra/views/editor/panel-selection-info.js

```diff
@@ -409,6 +428,42 @@ export default class SelectionInfoPanel extends Panel {
     }
   }
 
+  async _toggleGlyphLock(varGlyph) {
+    if (varGlyph.customData["fontra.glyph.locked"]) {
+      const result = await dialog(
+        `Are you sure you want to unlock glyph ${varGlyph.name}?`,
+        "",
+        [
+          { title: "Cancel", isCancelButton: true },
+          { title: "Yes", isDefaultButton: true, resultValue: "ok" },
+        ]
+      );
+
+      if (!result) {
+        // User cancelled
+        return;
+      }
+    }
+
+    await this.sceneController.editGlyphAndRecordChanges(
+      (glyph) => {
+        glyph.customData["fontra.glyph.locked"] =
+          !glyph.customData["fontra.glyph.locked"];
+
+        const iconElement =
```

Comment:
Please take this section out of the `editGlyphAndRecordChanges` section, and use the `varGlyph` variable you have available there.

Reply:
Done.

Reply:
Sorry. I changed it.